thrift/server.cc

```diff
@@ -362,6 +373,9 @@ thrift_stats::thrift_stats(thrift_server& server) {
                         sm::description(
                             seastar::format("Holds an incrementing counter with the requests that ever blocked due to reaching the memory quota limit ({}B). "
                                             "The first derivative of this value shows how often we block due to memory exhaustion in the \"Thrift transport\" component.", server.max_request_size()))),
+        sm::make_derive("requests_shed", [&server] { return server.requests_shed(); },
+                        sm::description("Holds an incrementing counter with the requests that were shed due to overload (threshold configured via max_concurrent_requests_per_shard). "
```

Comment:
The word "overload" is too overloaded :-) Instead of "due to overload" I think it will be clearer to say "due to a configured limit on the number of concurrent Thrift requests".
By the way, I put the word "Thirft" in this suggestion because, if I understand correctly (but please correct me if I'm wrong), the concurrency of different types of requests (CQL, Thrift, Alternator) is limited separately

Reply:
Alright, but in that case we should also reword the corresponding metrics in CQL (in a separate follow-up patch) - so that the two can be clearly distinguished as doing almost the same thing.